Source/Core/Core/HW/Memmap.cpp

```diff
@@ -391,7 +391,10 @@ u8* GetPointer(u32 address)
 {
   // TODO: Should we be masking off more bits here?  Can all devices access
```

Comment:
Yes. Testing is still needed. Though I'm pretty sure the answer is "All devices can access EXRAM"